lib/ansible/modules/cloud/openstack/os_volume.py

```diff
@@ -45,6 +45,9 @@
    volume:
      description:
        - Volume name or id to create from
+   bootable:
+     description:
+       - Bootable flag for volume
      version_added: "2.3"
```

Comment:
Additionally, looks like where `bootable` was added, it seems to have separated `version_added: "2.3"` from `volume`.